<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-05-15T20:36:00 IST" name="Default suite" finished-at="2022-05-15T20:36:12 IST" duration-ms="12183">
    <groups>
      <group name="negative">
        <method signature="DashboardPageTest.verifyAllTabsOnDashboardPage()[pri:0, instance:com.salesforce.qa.testdata.DashboardPageTest@478ee483]" name="verifyAllTabsOnDashboardPage" class="com.salesforce.qa.testdata.DashboardPageTest"/>
      </group> <!-- negative -->
      <group name="regression">
        <method signature="DashboardPageTest.verifyAllTabsOnDashboardPage()[pri:0, instance:com.salesforce.qa.testdata.DashboardPageTest@478ee483]" name="verifyAllTabsOnDashboardPage" class="com.salesforce.qa.testdata.DashboardPageTest"/>
      </group> <!-- regression -->
      <group name="sanity">
        <method signature="DashboardPageTest.verifyAllTabsOnDashboardPage()[pri:0, instance:com.salesforce.qa.testdata.DashboardPageTest@478ee483]" name="verifyAllTabsOnDashboardPage" class="com.salesforce.qa.testdata.DashboardPageTest"/>
      </group> <!-- sanity -->
    </groups>
    <test started-at="2022-05-15T20:36:00 IST" name="Default test" finished-at="2022-05-15T20:36:12 IST" duration-ms="12183">
      <class name="com.salesforce.qa.testdata.DashboardPageTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.salesforce.qa.testdata.DashboardPageTest@478ee483]" started-at="2022-05-15T20:36:00 IST" name="setUp" finished-at="2022-05-15T20:36:07 IST" duration-ms="7170" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifyAllTabsOnDashboardPage()[pri:0, instance:com.salesforce.qa.testdata.DashboardPageTest@478ee483]" started-at="2022-05-15T20:36:07 IST" name="verifyAllTabsOnDashboardPage" finished-at="2022-05-15T20:36:12 IST" duration-ms="4815" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Lists differ at element [11]: Note != Notes expected [Note] but found [Notes]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Lists differ at element [11]: Note != Notes expected [Note] but found [Notes]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.salesforce.qa.testdata.DashboardPageTest.verifyAllTabsOnDashboardPage(DashboardPageTest.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAllTabsOnDashboardPage -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.salesforce.qa.testdata.DashboardPageTest@478ee483]" started-at="2022-05-15T20:36:12 IST" name="tearDown" finished-at="2022-05-15T20:36:12 IST" duration-ms="156" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.salesforce.qa.testdata.DashboardPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
